drivers/bluetooth/hci/Kconfig

```diff
@@ -76,6 +76,8 @@ config BT_STM32WBA
 
 config BT_SILABS_HCI
 	bool "Silicon Labs Bluetooth interface"
+	default y
+	depends on DT_HAS_SILABS_BT_HCI_ENABLED
 	depends on SOC_SERIES_EFR32BG22 || SOC_SERIES_EFR32MG24 || SOC_SERIES_EFR32BG27
```

Comment:
If I understand right, `DT_HAS_SILABS_BT_HCI_ENABLED` should replace `SOC_SERIES_EFR32BG22 || SOC_SERIES_EFR32MG24 || SOC_SERIES_EFR32BG27`, shouldn't?

Reply:
> If I understand right, `DT_HAS_SILABS_BT_HCI_ENABLED` should replace `SOC_SERIES_EFR32BG22 || SOC_SERIES_EFR32MG24 || SOC_SERIES_EFR32BG27`, shouldn't?

Yes, that's a good point. Those should indeed be removed, thanks!